app/templates/suppliers/details.html

```diff
@@ -58,9 +58,11 @@
   {% call summary.row() %}
     {{ summary.field_name('Summary') }}
     {{ summary.text(supplier.description) }}
-    {% call summary.field(action=True) %}
-      Optional
-    {% endcall %}
+    {%  if not supplier.description %}
+      {% call summary.field(action=True) %}
+        Optional
+      {% endcall %}
+    {% endif %}
```

Comment:
I think we need to still generate an empty cell here or we'll get a line that terminates early.

Reply:
The line doesn't end early now, but it might if we had another "Optional" field. Do you think I should add empty field for consistency?


![screen shot 2018-02-28 at 15 14 28](https://user-images.githubusercontent.com/20957548/36795172-3cb8b456-1c9a-11e8-981f-d810c67c98f4.png)

Reply:
Yep! Could catch someone out if we try to add another row in the future. Good to prevent that if we can.